core/src/main/java/com/hcl/ecomm/core/services/impl/CustomSendEmailWorkflowProcess.java

```diff
@@ -97,7 +97,13 @@ public void execute(WorkItem workItem, WorkflowSession workflowSession, MetaData
 
             Map<String, String> params = new HashMap<>();
             InternetAddress[] iAddressRecipients = addresses.toArray(new InternetAddress[addresses.size()]);
-            List<InternetAddress> failureInternetAddresses = emailService.sendEmail(templatePath, params, attachments, iAddressRecipients);
+            if(attachments.isEmpty())
```

Comment:
@DeepaliAgrwl  Why we need attachment here?

Reply:
I just reverted back this change. Please verify